package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@pyviz/jupyterlab_pyviz",
-  "version": "0.7.1",
+  "version": "1.0.0-alpha.6",
```

Our versioning does not mirror the JupyterLab versioning so let's go with:

```suggestion
  "version": "0.8.0-alpha.1",
```